setup.py

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+try:
+    from distutils.core import setup
+except ImportError:
+    from setuptools import setup
```

Comment:
I think this should just import `setuptools` and use conditional dependencies

https://github.com/ros-controls/ros_control/pull/429/files#diff-f3ba5e3bc4f4fc2a8ce3f52bfc533377

Reply:
format 3 with conditional dependencies in 9de2aa8 and setuptools in e2ad177